client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -167,7 +168,7 @@ const LoggedOutForm = React.createClass( {
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ login( { redirectTo } ) }>
-					{ this.translate( 'Already have an account? Sign in' ) }
+					{ i18n.translate( 'Already have an account? Sign in' ) }
```

Comment:
Would it not work to change this from `this.translate` to `this.props.translate` here?